metrotranscom/doorway
# refactor: remove grouped table from doorway-uic

use standardTable instead

# Pull Request Template

## Issue Overview

This PR addresses [#46](https://github.com/bloom-housing/Bloom-Doorway/issues/46)


## Description

Removes GroupedTable to use StandardTable instead.

## How Can This Be Tested/Reviewed?

On public site go to listing. First table (tables) that are related to ami will now use StandardTable, it shouldn' change anything visually.
Check for single ami value, and for multiple ami values (just check few different listings)

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
